### PR TITLE
SUS-2956: Migrate MultiLookup updates to MediaWiki task

### PR DESCRIPTION
**Pull request by mszabo-wikia**
https://wikia-inc.atlassian.net/browse/SUS-2956

### CODE REVIEW

File: extensions/wikia/SpecialMultipleLookup/MultiLookupHooks.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		$ip = $recentChange->getAttribute( 'rc_ip' );
+
+		if ( IP::isIPAddress( $ip ) ) {
+			global $wgCityId;
+
+			$task = ( new \Wikia\Tasks\Tasks\MultiLookupTask() )
+				->wikiId( $wgCityId );
+			$task->call( 'updateMultiLookup', $ip );
+			$task->queue();
+		}
```
Comment (macbre) [inline extensions/wikia/SpecialMultipleLookup/MultiLookupHooks.php:33]: Does it make sense to log cases when `$ip` is not a proper IPv4/IPv6?

**Review by macbre — APPROVED** [review]
As discussed offline, let's make sure that `SpecialMultipleLookup.php` is enabled site-wide and move the `include` to app's `DefaultSettings.php` file.

File: extensions/wikia/SpecialMultipleLookup/MultiLookupHooks.php (hunk 1 above)
Comment (TK-999) [reply extensions/wikia/SpecialMultipleLookup/MultiLookupHooks.php]: If IP is not valid, we do not queue task here

Comment (mszabo-wikia) [reply extensions/wikia/SpecialMultipleLookup/MultiLookupHooks.php:33]: It can happen if some wiki has `wgPutIPinRC = false;`, unlikely, but possible.

**Pull request merged**